extensions/blocks/slideshow/slideshow.php

```diff
@@ -23,12 +23,7 @@
  * @return string
  */
 function jetpack_slideshow_block_load_assets( $attr, $content ) {
-	$dependencies = array(
-		'lodash',
-		'wp-element',
-		'wp-i18n',
```

Comment:
Ah okay, we're not directly importing from `@wordpress/i18n` in the Slideshow block but are using the [wrapper](https://github.com/Automattic/wp-calypso/blob/71ad326afeb9fcf9b4e05bd6840b248ea3c8e655/client/gutenberg/extensions/slideshow/index.js#L5) which does that.

Makes sense -- it just means we'll have to add the dependency back here once we've got rid of the wrapper.